docs/pages/getting-started/overview.md

```diff
@@ -1,8 +1,8 @@
-<!-- ---
```

Comment:
Here also,
<--- ---

Remove only `<---` and the other 3 --- should exist

Reply:
done